hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestDefaultStoreEngine.java

```diff
@@ -67,6 +67,7 @@ public void testCustomParts() throws Exception {
         DummyStoreFlusher.class.getName());
     HRegion mockRegion = Mockito.mock(HRegion.class);
     HStore mockStore = Mockito.mock(HStore.class);
+    mockStore.conf = conf;
```

Comment:
This is a bit strange...
We can not do something like `when(mockStore.getConf()).thenReturn(conf);` ?

Reply:
There's no getter for `conf` in HStore.

Reply:
OK, can leave it for now. Could open new issues to change it later. Not a blocker.